modules/tinymce/src/models/dom/main/ts/table/core/Util.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Similar to my previous comment this probably should be called `Utils` instead of `Util`. Also since there will be more in the model eventually I'd suggest calling it `TableUtils` like in core.